x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/min_age_input.tsx

```diff
@@ -68,9 +68,23 @@ function getUnitsAriaLabelForPhase(phase) {
   }
 }
 
-export const MinAgeInput = (props) => {
-  const { rolloverEnabled, errors, phaseData, phase, setPhaseData, isShowingErrors } = props;
+interface Props {
+  rolloverEnabled: boolean;
+  errors: Record<string, string[]>;
+  phaseData: any;
+  phase: string;
+  setPhaseData: (dataKey: string, value: any) => void;
```

Comment:
Is there some way that we could get typings for this `any`? Perhaps just for the things the current code expects to be there?

Reply:
If not that approach then just a `TODO:` comment would be useful

Reply:
good idea with a `TODO`, added those comments to fix in the next PR